internal/system/agent/const.go

```diff
@@ -29,6 +29,11 @@ type ConfigurationStruct struct {
 	OsLevelOperations     bool
 	DockerLevelOperations bool
 	Clients               map[string]config.ClientInfo
+	Databases             map[string]config.DatabaseInfo
+	Logging               config.LoggingInfo
+	Notifications         config.NotificationInfo
+	Registry              config.RegistryInfo
+	Service               config.ServiceInfo
```

Comment:
I removed these as part of PR #668 because they weren't used. The configuration.toml isn't structured in a way as to take advantage of these elements.

Reply:
👍